website/src/pages/_app.tsx

```diff
@@ -0,0 +1,17 @@
+import { createBrowserSupabaseClient } from "@supabase/auth-helpers-nextjs";
+import { SessionContextProvider } from "@supabase/auth-helpers-react";
+import { useState } from "react";
+
+import "../styles/globals.css";
+import "focus-visible";
+
+function MyApp({ Component, pageProps }) {
+  const [supabase] = useState(() => createBrowserSupabaseClient());
+  return (
+    <SessionContextProvider supabaseClient={supabase} initialSession={pageProps.initialSession}>
```

Comment:
Can you switch this to use the next auth session provider?

Reply:
Certainly